test/nn/test_embedding.py

```diff
@@ -1002,9 +1003,9 @@ def _test_EmbeddingBag_vs_Embedding(self, N, D, B, L, max_norm=None,
 
         output.backward(grad_output)
         ref_output.backward(grad_output)
-        es_weight_grad = es.weight.grad.data
+        es_weight_grad = copy.deepcopy(es.weight.grad)
```

Comment:
```suggestion
        es_weight_grad = es.weight.grad.detach()
```

or use clone()?  deepcopy is likely not the right thing to do :p

Reply:
ok done